test/encryption/ssss_test.dart

```diff
@@ -107,9 +107,15 @@ void main() {
       await handle.unlock(recoveryKey: ssssKey);
       expect(handle.isUnlocked, true);
       FakeMatrixApi.calledEndpoints.clear();
-      await handle.store('best animal', 'foxies');
+
+      // OpenSSSS store waits for accountdata to be updated before returning
+      // but we can't update that before the below endpoint is not hit.
+      await handle.ssss
+          .store('best animal', 'foxies', handle.keyId, handle.privateKey!);
       // alright, since we don't properly sync we will manually have to update
       // account_data for this test
+
+      //   await handle.store('best animal', 'foxies');
```

Comment:
Should this get removed now or what is this additional commented out code for? :3